app/controllers/occupation_standards_controller.rb

```diff
@@ -1,11 +1,24 @@
 class OccupationStandardsController < ApplicationController
   def index
-    @pagy, @occupation_standards = pagy(
-      OccupationStandard.includes(:organization, occupation: :onet)
+    @occupation_standards_search = OccupationStandardQuery::Container.new(search_term: search_term_params)
+
+    @occupation_standards = OccupationStandardQuery.run(
+      OccupationStandard.includes(:organization, occupation: :onet),
+      search_term_params
     )
+
+    @pagy, @occupation_standards = pagy(@occupation_standards)
   end
 
   def show
     @occupation_standard = OccupationStandard.find(params[:id])
   end
+
+  private
+
+  def search_term_params
+    {
+      search_term: params[:q]
```

Comment:
What do you think about using `q` here as the key also? I'm kind of torn. I like `search_term` because it is a well-named variable, but I feel like the `search_term[:search_term]` for the Struct gets a little confusing.

Reply:
@littleforest makes sense. Updated